src/utils/OpenSwathWorkflow.cpp

```diff
@@ -467,8 +467,11 @@ class TOPPOpenSwathWorkflow
     registerInputFile_("swath_windows_file", "<file>", "", "Optional, tab-separated file containing the SWATH windows for extraction: lower_offset upper_offset. Note that the first line is a header and will be skipped.", false, true);
     registerFlag_("sort_swath_maps", "Sort input SWATH files when matching to SWATH windows from swath_windows_file", true);
 
-    registerFlag_("use_ms1_traces", "Extract the precursor ion trace(s) and use for scoring", true);
-    registerFlag_("enable_uis_scoring", "Enable additional scoring of identification assays", true);
+    registerStringOption_("use_ms1_traces", "<name>", "true", "Extract the precursor ion trace(s) and use for scoring if present", false, true);
+    setValidStrings_("use_ms1_traces", ListUtils::create<String>("true,false"));
```

Comment:
@grosenberger can you look at these parameter settings and cross-check with your settings please